.github/workflows/release.yaml

```diff
@@ -0,0 +1,216 @@
+---
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The new version to release. E.g. `1.2.3`"
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  BAZEL_STARTUP_FLAGS: --bazelrc=${{ github.workspace }}/.github/github.bazelrc
+
+jobs:
+  validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: main
+      - name: Detect the current version
+        run: |
+          version="$(grep 'VERSION =' ${{ github.workspace }}/version.bzl | sed 's/VERSION = "//' | sed 's/"//')"
```

Comment:
As mentioned below, you can replace two seds with a single grep

Reply:
Oh I maybe figured it out. Is the release workflow as follows?

1) I update the constant in version.bzl 
2) I run the github action with the same updated version as input?

What's the motivation for the two step process? Would it be an option to not have action input and read the version number from the bzl file? To prevent human error we could somehow check if a release for the given version was already created and fail the release if yes.

Reply:
I've changed some things around, let me know what you think!